aws/policy/compute.yaml

```diff
@@ -174,17 +182,13 @@ Statement:
       - elasticloadbalancing:SetLoadBalancerPoliciesOfListener
       - elasticloadbalancing:SetSecurityGroups
       - elasticloadbalancing:SetWebACL
-      - elasticfilesystem:CreateFileSystem
-      - elasticfilesystem:CreateMountTarget
-      - elasticfilesystem:CreateTags
-      - elasticfilesystem:DeleteFileSystem
-      - elasticfilesystem:DeleteMountTarget
-      - elasticfilesystem:Describe*
-      - elasticfilesystem:ListTagsForResource
-      - elasticfilesystem:PutLifecycleConfiguration
-      - elasticfilesystem:TagResource
-      - elasticfilesystem:UntagResource
-      - elasticfilesystem:UpdateFileSystem
+      - elasticloadbalancing:DescribeInstanceHealth
+      - elasticloadbalancing:DescribeLoadBalancerAttributes
+      - elasticloadbalancing:DescribeLoadBalancerPolicies
+      - elasticloadbalancing:DescribeLoadBalancers
+      - elasticloadbalancing:DescribeTargetGroupAttributes
+      - elasticloadbalancing:DescribeTargetGroups
+      - sts:GetCallerIdentity
```

Comment:
What requires GetCallerIdentity here?

Reply:
This seem to be using ```GetCallerIdentity.```
https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_asg/tasks/main.yml#L93

Reply:
Removed as discussed.